tmt/steps/discover/fmf.py

```diff
@@ -91,9 +96,9 @@ def options(cls, how=None):
 
     def show(self):
```

Comment:
A bit unrelated note, this `show` should also have the `options` argument as per the [Liskov Substitution Principle](https://stackify.com/solid-design-liskov-substitution-principle/) (you should be able to replace the call to the parent with a call to the children which is not the case here due to the different arguments). I've seen this at a few places already, fixed it in some of them so far so it would be great to fix it everywhere :)

Reply:
Yes, sorry, bad argument name. The same goes for `wake()` which has `options` argument so that probably confused me a bit